src/main/java/Duke.java

```diff
@@ -1,10 +1,64 @@
+import duke.command.Command;
+
+import duke.exception.DukeException;
+
+import duke.util.Parser;
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
```

Comment:
I never realized my classes could be organised this way (though in hindsight, it does seem kind of obvious). Thanks for the inspiration.